src/web/services/http-request.service.ts

```diff
@@ -1,8 +1,42 @@
-import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+import { HttpClient, HttpHeaders, HttpParameterCodec, HttpParams } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { environment } from '../environments/environment';
 
+/**
+ * Custom HttpParameter encoder
```

Comment:
At its state right now, the doc is redundant. The least you can do it mention how it is different and/or why a custom encoder/decoder is necessary.